app/views/concerns/navigable_view.rb

```diff
@@ -138,6 +140,23 @@ def navigation_global_primary_nav_blog_submenu_items
                             is_morelink: true)
   end
 
+  def navigation_global_primary_nav_galleries
+    {
+      text: t('global.navigation.galleries'),
+      is_current: controller.controller_name == 'galleries',
+      submenu: {
+        items: navigation_global_primary_nav_galleries_submenu_items
+      }
+    }
+  end
+
+  def navigation_global_primary_nav_galleries_submenu_items
+    Gallery.published.order(published_at: :desc).map do |gallery|
```

Comment:
This will include **all** published galleries in the submenu which will soon become unusable. It should be limited to the _x_ most recent.